packages/beacon-dapp/src/dapp-client/DAppClient.ts

```diff
@@ -1680,7 +1684,7 @@ export class DAppClient extends Client {
       }
 
       return {
-        name: walletInfo.name ?? selectedApp?.name,
+        name: selectedApp?.name ?? walletInfo.name,
```

Comment:
Reading from storage we get the wallet key and not the wallet name (i.g. inside the toast 'kukai_ios' will be displayed instead of 'Kukai Wallet'